### DISCUSSION TITLE
Can you support export docx？

### DISCUSSION THREAD

**Discussion by Adonis0123** [toeverything/blocksuite#3003] [Q&A]
My current approach involves using block2Html to obtain an HTML string, which I then export as a DOCX file using html-docx-js-typescript. Can Blocksuite directly support this functionality? Thank you.

**Comment by doodlewind** [comment]
Sure, supporting custom exporters is on our plan, but this may require some more time to make this happen.

**Reply by junoriosity** [reply]
@doodlewind This is very interesting to me as well. Are there any updates on that?

**Reply by Adonis0123** [reply]
here is my code，I don't know if it'll help you

```
import saveAs from 'file-saver'
import { asBlob } from 'html-docx-js-typescript'

export default async function handleExportDocx({
  title,
  content,
}: {
  title: string
  content: string
}) {
  const blobData = (await asBlob(content, {
    orientation: 'landscape',
  })) as Blob

  saveAs(blobData, `${title}.docx`)
}

```
